docs/user-import.md

```diff
@@ -0,0 +1,67 @@
+# Importing users into identity
+
+## Overview
+
+As more and more DfE services migrate to using `Identity` to manage authentication and authorisation, there is the need to migrate existing users of these services into identity.  
+While this could be done the next time the user interacts with the specific service, it would mean them having to re-enter their details in order to create a teaching account.  
+The user import feature has been created to automate the creation of teaching accounts in identity from users of other DfE services as much as possible.
+
+## Import File Definition
+
+The user import requires a CSV file with a header with the following fields:
```

Comment:
Worth calling out that all fields are mandatory except TRN?

Reply:
Yeah I thought I alluded to that with the "Empty" being a valid option but .. and technically you still need it in the CSV even if it's empty

Reply:
Added a column to the table to indicate the mandatoryness of the fields